pkg/v1/providers/ytt/03_customizations/02_avi/add_akoo.yaml

```diff
@@ -2,15 +2,15 @@
 #@ load("@ytt:json", "json")
 #@ load("static.lib.yaml", "crd","service")
 #@ load("rbac.lib.yaml", "role", "role_binding")
-#@ load("ako-operator-deployment.lib.yaml", "akoo_deployment", "ako_deployment_config", "avi_credentials", "avi_certificate")
+#@ load("ako-operator-deployment.lib.yaml", "akoo_deployment", "avi_credentials", "avi_certificate")
 
+#! render the ako-operator artifacts
 #@ if data.values.PROVIDER_TYPE == "vsphere" and data.values.TKG_CLUSTER_ROLE != "workload" and data.values.AVI_ENABLE and data.values.AVI_CONTROL_PLANE_HA_PROVIDER:
 --- #@ crd()
```

Comment:
keep this crd for now. Current ako operator will crash if this crd were not found.